### PR TITLE
Updated Faq

### PR DESCRIPTION
**Pull request by nassaleh**
Issue: #
https://nventive.visualstudio.com/Umbrella/_workitems/edit/134056

## PR Type
What kind of change does this PR introduce?

Documentation content changes 

## What is the new behavior?
Updated documentation

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes

### CODE REVIEW

File: doc/articles/faq.md [hunk 1]
```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
...
+
+It's in the UWP API, but [not implemented yet](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
+
+## Is there a table of all UWP controls and their features compared to what's offered under Uno?
+
+https://github.com/nventive/Uno/blob/master/doc/articles/supported-features.md
+
+##  Is there an Uno template that is based on portable class library?
+
+https://github.com/nventive/Uno/pull/132
```
Comment (jeromelaban) [inline doc/articles/faq.md]: Should be: _No, but use the [Cross-Platform library template](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin) instead to achieve a similar goal.

File: doc/articles/faq.md [hunk 2]
```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
...
+If you are using JSON.NET, you need [this](https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.WASM/LinkerConfig.xml) 
+This file is referenced in the .csproj like [that](https://github.com/nventive/Uno.Playground/blob/master/src/Uno.Playground.WASM/Uno.Playground.WASM.csproj#L43)
+
+## Is NavigationView supported in Uno?
+
+It is indeed not implemented at the time, see [nventive/Uno#4](https://github.com/nventive/Uno/issues/4) for now
+
+## Is there any particular reason that Uno uses a shared project? and is it possible to use a netstandard project instead?
+
+The view layer needs to be in a shared project because it has native dependencies.
```
Comment (jeromelaban) [inline doc/articles/faq.md]: You can add: _Shared projects are also used to work around the fact that Visual Studio is not able to selectively compile a single Target Framework when building a cross-targeted library. Using a shared project improves the build performance when debugging a single platform._

File: doc/articles/faq.md [hunk 3]
```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
...
+
+Not yet.
+
+## Is there a way to use local css/js libraries and not those on a CDN?
+
+you can specify a custom HTML template like [this](https://github.com/nventive/Uno.Wasm.Bootstrap#indexhtml-content-override)
+
+## Debugging a published nuget package breaks the Xamarin iOS and Android debugger
+
+Please see [here](https://developercommunity.visualstudio.com/content/problem/306764/debugging-a-published-nuget-package-break-the-xama.html)
```
Comment (jeromelaban) [inline doc/articles/faq.md]: You can add, _this has been fixed starting from Visual Studio 15.9 Preview 3_

File: doc/articles/faq.md [hunk 4]
```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
...
+
+## Does Uno offer an `AutoSuggestBox`?
+
+[Yes](https://github.com/nventive/Uno/blob/master/src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs)
+
+## Is there a database that works on all platforms?
+
+SQLite should work on all platforms (except Wasm) https://www.nuget.org/packages/sqlite-net-pcl/
```
Comment (jeromelaban) [inline doc/articles/faq.md]: Change this, _Yes, and you can use this WebAssembly SQLite provider:  https://github.com/nventive/Uno.SQLitePCLRaw.Wasm_

File: doc/articles/faq.md [hunk 5]
```diff
@@ -286,3 +286,83 @@ Yes, make sure to use the following project definition:
     <Reference Include="System.Runtime.Serialization" />
 </ItemGroup>
 ```
+
+## Does Uno offer a TreeView control?
+
+It's in the UWP API, but [not implemented yet](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
```
Comment (jeromelaban) [inline doc/articles/faq.md]: Mention this issue: https://github.com/nventive/Uno/issues/3

File: doc/articles/faq.md [hunk 6]
```diff
@@ -286,3 +286,87 @@ Yes, make sure to use the following project definition:
...
+We have an open Github (issue.)[https://github.com/nventive/Uno/issues/3]
+
+## Is there a table of all UWP controls and their features compared to what's offered under Uno?
```
Comment (davidjohnoliver) [inline doc/articles/faq.md]: 'a table of all UWP controls' - @jeromelaban what's the status of the generated web pages? Are they going to see the light?

Comment (jeromelaban) [reply doc/articles/faq.md]: They will, the doc site is being looked at currently.

**Pull request merged**